src/gui/src/components/explorer-grid/dependency-side-bar.tsx

```diff
@@ -232,14 +231,11 @@ export const DependencySideBar = ({
                 <PopoverTrigger>
                   <Tooltip>
                     <TooltipTrigger asChild>
```

Comment:
it may be possible to remove `asChild` from here now, need to take a look at the console logs to be sure though.

```suggestion
                    <TooltipTrigger>
```

Reply:
Looks like we'll need to leave them in to avoid the console logs